support/docker/docker-compose.yml

```diff
@@ -0,0 +1,13 @@
+version: '3'
+
+services:
+  netpanzer-server:
+    image: andy5995/netpanzer:latest
+    container_name: netpanzer-server
+    ports:
+      - "3031:3031/tcp"
+      - "3031:3031/udp"
+    command: su - npserver -c "cd /netpanzer && ./netpanzer -d"
+    volumes:
+      - ./opt:/home/npserver/.netpanzer
```

Comment:
docker will mount the local "opt" directory in the container as /home/npserver/.netpanzer and relative to that is the config directory and server.cfg, as it normally would be.